layouts/DocsPage/types.ts

```diff
@@ -3,17 +3,42 @@
 import type { IconName } from "components/Icon/types";
 import { VideoBarProps } from "components/VideoBar/types";
 
-export const scopeValues = ["oss", "cloud", "enterprise"] as const;
+export const scopeValues = ["oss", "enterprise", "cloud"] as const;
 
 export type ScopeType = typeof scopeValues[number];
 export type ScopesType = ScopeType | ScopeType[];
 
-export interface NavigationItem {
+export type ComplexScopesConfig =
```

Comment:
Is this because you don't enforce an order in the frontmatter?

Reply:
@Alqanar I've merged, but can you please address some questions and possibly fixes if necessary in the follow up PRs?

Reply:
@hatched Not really. If it were specified in frontmatter, someone would have to specify scopes for navigation in config as well. Because navigation doesn't get information from frontmatter.